tests/testthat/testSuite004.01-end2end-testIncidence-state2.1changes.R

```diff
@@ -70,12 +70,12 @@ error = function(e) {
 
 test_that("Test 1 output matches", {
   # Compare the output to EpiEstim's output
-  I <- read.csv(paste(appDir, 'datasets/IncidenceData/H1N1Pennsylvania2009.csv', sep='/'), header=FALSE)
-  I <- EpiEstim:::process_I(I)
```

Comment:
Not for this PR, but probably `process_I` should be renamed to `process_incid` inside EpiEstim?